lms/static/scripts/frontend_apps/services/index.js

```diff
@@ -3,6 +3,7 @@ import { useContext, useMemo } from 'preact/hooks';
 
 export { ClientRpc } from './client-rpc';
 export { GradingService } from './grading';
+export { VitalSourceService } from './vitalsource';
```

Comment:
I wonder if there is a way to prevent future devs from forgetting to add an `export` here when adding a new service. I could see it happening...

Reply:
Hopefully developers will notice when they try to register the service in `frontend_apps/index.js` or use it in a component, following the example of existing services, and find the import missing.